### PR TITLE
Update snapshots for changes in upstream

### PR DESCRIPTION
**Pull request by IndrajeetPatil**

### CODE REVIEW

File: DESCRIPTION [hunk 1]
```diff
@@ -19,14 +19,14 @@ License: MIT + file LICENSE
 URL: https://pillar.r-lib.org/, https://github.com/r-lib/pillar
 BugReports: https://github.com/r-lib/pillar/issues
 Imports: 
-    cli (>= 2.3.0),
+    cli (>= 3.2.0),
```
Comment (IndrajeetPatil) [inline DESCRIPTION:22]: Because `{vctrs}` needs this version: https://cran.r-project.org/web/packages/vctrs/index.html

**Comment by IndrajeetPatil** [comment]
@krlmlr All builds passing now.

File: tests/testthat/_snaps/format_integer64.md [hunk 2]
```diff
@@ -98,66 +98,66 @@
       pillar(x, width = 16)
     Output
       <pillar>
-      <nanotime>      
+      <int64>
```
Comment (krlmlr) [inline tests/testthat/_snaps/format_integer64.md:101]: This is wrong. Where does it come from?

File: DESCRIPTION (hunk 1 above)
Comment (krlmlr) [inline DESCRIPTION:22]: Do we really want to update versions in `DESCRIPTION` just to satisfy snapshot tests?

File: tests/testthat/_snaps/format_integer64.md (hunk 2 above)
Comment (IndrajeetPatil) [reply tests/testthat/_snaps/format_integer64.md:101]: From `{bit64}`:

``` r
library(pillar)
suppressPackageStartupMessages(library(bit64))

x <- as.integer64(c(0, 1e11, 1e12) + 0:2)
pillar(x)
#> <pillar>
#>       <int64>
#>             0
#>  100000000001
#> 1000000000002
```

<sup>Created on 2022-10-16 with [reprex v2.0.2](https://reprex.tidyverse.org)</sup>

File: DESCRIPTION (hunk 1 above)
Comment (IndrajeetPatil) [reply DESCRIPTION]: Reverted versions bumps.

**Comment by krlmlr** [comment]
The original snapshot is correct. Can we find out when this problem first occurred? Or should we focus on the "why"?

**Comment by krlmlr** [comment]
I wonder if this gets better with dev vctrs.

**Comment by IndrajeetPatil** [comment]
> I wonder if this gets better with dev vctrs.

You are right! With devel-`{vctrs}`, the snapshots are as expected.

**Comment by krlmlr** [comment]
Thanks!

**Pull request merged**